help/de/Help_ch_GUIMenus.xml

```diff
@@ -5796,9 +5787,11 @@ url="https://www.gnucash.org/docs/v&series-stable;/C/gnucash-guide/chapter_txns.
         verwendet werden.
       </para>
 
-      <para>Eine kurze Beschreibung der Funktion einer <emphasis>Werkzeug</emphasis>-Schaltfläche wird
-        angezeigt, wenn der Mauszeiger für einige Sekunden über dem Symbol platziert wird.
-      </para>
+      <tip>
+        <para>Eine kurze Beschreibung der Funktion einer <emphasis>Werkzeug</emphasis>-Schaltfläche wird
+          angezeigt, wenn der Mauszeiger für einige Sekunden über dem Symbol platziert wird.
+        </para>
+      </tip>
```

Comment:
Sooft der Text vorkommt, wäre er eventuell besser in einer [`<footnote>`](https://tdg.docbook.org/tdg/4.5/footnote.html) untergebracht. 
Entität wäre auch denkbar, oder
reduziert sich die Anzahl bei der angedachten Neugliederung?